### PR TITLE
Add Arabic Egpyt Translation & Add Impostor Radio and other things to Translations & Fix Grammar in GitHub Actions

### PR DESCRIPTION
**Pull request by MatadorProBr**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -11,10 +11,10 @@ jobs:
         include:
          - os: windows-latest
            dist: 'dist'
-           installer: 'dist/Better-CrewLink Setup*.exe'
```
Comment (OhMyGuus) [inline .github/workflows/build.yml:14]: You can't change the installer name here, that is based of the electron-builder.yml but changing things there will frick the updater.

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -11,10 +11,10 @@ jobs:
...
+           installer: 'dist/BetterCrewLink Setup*.exe'
          - os: ubuntu-latest
            dist: 'dist:linux'
-           installer: 'dist/Better-CrewLink-*.AppImage'
```
Comment (OhMyGuus) [inline .github/workflows/build.yml:17]: You can't change the installer name here, that is based of the electron-builder.yml but changing things there will frick the updater.

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -11,7 +11,7 @@ jobs:
         include:
          - os: windows-latest
            dist: 'publish'
-           installer: 'dist/Better-CrewLink Setup*.exe'
```
Comment (OhMyGuus) [inline .github/workflows/release.yml:14]: Same here

File: appveyor.yml [hunk 4]
```diff
@@ -26,17 +26,17 @@ on_failure:
 
 test_script:
   - ps: |
-      if(!(ls dist\Better-Crewlink*))
+      if(!(ls dist\BetterCrewLink*))
```
Comment (OhMyGuus) [inline appveyor.yml:29]: same here..

File: appveyor.yml [hunk 5]
```diff
@@ -26,17 +26,17 @@ on_failure:
...
       {
        exit 1
       }
 artifacts:
   - path: dist/win-unpacked
     name: Unpacked
     type: auto
-  - path: 'dist/Better-CrewLink Setup*.exe'
-    name: installer
+  - path: 'dist/BetterCrewLink Setup*.exe'
```
Comment (OhMyGuus) [inline appveyor.yml:37]: same here..

File: appveyor.yml [hunk 6]
```diff
@@ -26,17 +26,17 @@ on_failure:
...
+    name: Installer
     type: auto
-  - path: 'dist/Better-CrewLink-*.AppImage'
```
Comment (OhMyGuus) [inline appveyor.yml:40]: same here..

**Review by OhMyGuus — CHANGES_REQUESTED** [review]
You can't change the installer name here, that is based of the electron-builder.yml but changing things there will frick the updater.

**Comment by MatadorProBr** [comment]
Probably your answer will be no but can I change that?
https://github.com/MatadorProBr/BetterCrewLink/blob/nightly/electron-builder.yml#L1

**Comment by OhMyGuus** [comment]
> appId: net.ohmyguus.bettercrewlink



> Probably your answer will be no but can I change that?
> https://github.com/MatadorProBr/BetterCrewLink/blob/nightly/electron-builder.yml#L1
> 
> Change of:
> appId: net.ottomated.crewlink.beta
> 
> to:
> appId: net.ohmyguus.bettercrewlink
> 
> or anything like this ^

Nope, maybe later but that needs more testing in a private repo

**Review by OhMyGuus — APPROVED** [review]

**Pull request merged**